daisy_workflows/image_build/windows/post_install.ps1

```diff
@@ -359,6 +359,43 @@ function Set-Repos {
   Run-Command 'C:\ProgramData\GooGet\googet.exe' -root 'C:\ProgramData\GooGet' addrepo 'google-compute-engine-stable' 'https://packages.cloud.google.com/yuck/repos/google-compute-engine-stable'
 }
 
+function Export-ImageMetadata {
+  $version = (Get-ComputerInfo).OsVersion
+  $family = 'windows-' + (Get-ComputerInfo).windowsversion
+  $name =  (Get-ComputerInfo).OSName
+  $release_date = (Get-Date).ToUniversalTime()
+  $image_metadata = @{'family' = $family;
+                      'version' = $edition;
+                      'name' = $name;
+                      'location' = ${script:outs_dir};
+                      'build_date' = $release_date;
+                      'build_repo' = $stage;
+                      'packages' = @()}
+
+  # Get Googet packages.
+  $out = & 'C:\ProgramData\GooGet\googet.exe' -root 'C:\ProgramData\GooGet' 'installed'
+  $out = $out[1..$out.length]
+  [array]::sort($out)
+
+  foreach ($package_line in $out) {
+    $split = $package_line.Trim() -split '\s+'
+    $name = $split[0]
+    $version = $split[1]
+    # TODO: Currently, "Googet Installed" command only return name and version.
+    # We need to update googet command to get all info.
+    $commit_hash = ""
+    $package_metadata = @{'name' = $name;
+                          'version' = $version;
+                          'commmit_hash' = $commit_hash}
+    $image_metadata['packages'] += $package_metadata
+  }
+
+  # Save the JSON image_metadata.
+  $image_metadata_json = $image_metadata | ConvertTo-Json -Compress
+  $image_metadata_json | & 'gsutil' -m cp - "${script:outs_dir}/metadata.json"
```

Comment:
Would it be helpful to log this as well? Or would it be too noisy?

Reply:
I would not expect to log the json as it's not critical path for the windows build (wrong format affect nothing related building)